seedlet/assets/sass/blocks/file/_style.scss

```diff
@@ -9,10 +8,17 @@
 		opacity: inherit;
 	}
 
+	a:after {
+		content: '';
+		display: block;
+	}
+
 	.wp-block-file__button {
 		// Extend button style
-		@extend %button-style;
+		@include button-style;
```

Comment:
When this is an `extend` the padding declaration below doesn't work. That's why we need a mixin.